cfme/fixtures/smtp.py

```diff
@@ -70,6 +70,12 @@ def _finalize():
                 collector.send_signal(signal.SIGKILL)
         collector.wait()
         logger.info("Collector finished")
+        logger.info("Cleaning up smtp setup in CFME")
+        smtp_conf = configuration.SMTPSettings(
```

Comment:
cleaning up smtp settings. other way could be just scope change to 'module' for smtp_test fixture which could save some time too